skin-resque-frontend/src/components/Heading/Heading.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './Heading.scss'
+
+export default function Heading({size, children}){
+    const tags = {
+        "x-large": "h1",
+        "large": "h2",
+        "medium": "h3",
+        "small": "h4"
+    }
+    const Tag = tags[size]
```

Comment:
Couldn't pull your branch, but are you sure this will work? To me it looks like the type would be string and not () => ReactComponent. Did you test this?

Reply:
Yes, it works